Developers/Game #2/Assets/Player_Jump.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Player_Jump : MonoBehaviour
+{
+
+    public float jumpForce;
+    private float moveInput;
+    private Rigidbody2D rb;
+
+    private bool isGrounded;
+    public Transform feetpos;
+    public float checkRadius;
+    public LayerMask WhatIsGround;
+
+    private float jumpTimeCounter;
+    public float jumpTime;
+    private bool isJumping;
+    // Start is called before the first frame update
+    void Start()
+    {
+        rb = GetComponent<Rigidbody2D>();

```

Comment:
I added `rb = GetComponent<Rigidbody2D>();`. This is where your null exception was coming from. Whenever you declare a component in your script you want to use `GetComponent<T>()` to get it.